apps/web/src/pages/studio-onboarding/success.tsx

```diff
@@ -0,0 +1,82 @@
+import { Title, Text } from '@novu/novui';
+import { css } from '@novu/novui/css';
+import { VStack } from '@novu/novui/jsx';
+import { useQuery } from '@tanstack/react-query';
+import { useMemo } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { getActivityList } from '../../api/activity';
+import { ExecutionDetailsAccordion } from '../../components/execution-detail/ExecutionDetailsAccordion';
+import { ROUTES } from '../../constants/routes';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+
+export const StudioOnboardingSuccess = () => {
+  const { data } = useQuery<{ data: any[]; hasMore: boolean; pageSize: number }>(['activitiesList', 0], () =>
+    getActivityList(0, undefined)
+  );
+
+  const item = useMemo(() => {
+    if (!data) {
+      return null;
+    }
+
+    return data.data.at(-1);
+  }, [data]);
+
+  const identifier = useMemo(() => {
+    if (item === null) {
+      return undefined;
+    }
+
+    return item?.trigger?.identifier;
+  }, [item]);
+
+  const email = useMemo(() => {
+    if (item === null) {
+      return null;
+    }
+
+    return item.subscriber.email;
+  }, [item]);
+
+  const navigate = useNavigate();
+
+  return (
+    <div
+      className={css({
+        width: '100dvw',
+        height: '100dvh',
+      })}
+    >
+      <Header active={3} />
+      <VStack alignContent="center">
+        <div
+          className={css({
+            width: '880px',
+          })}
+        >
+          <Title variant="section">Success, you sent an email to {email}</Title>
+          <Text
+            className={css({
+              color: 'typography.text.secondary',
+              lineHeight: '125',
```

Comment:
```suggestion
            variant="secondary"
            className={css({
```

Reply:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** If you find yourself having to set line height, there's a good chance that:

1. There's a pre-built utility that includes it
2. We need to talk to Nikolay since he may be not staying within Design System definitions 👮 

1. ‏

Reply:
I think it was just missed by me to remove it :(